input-output-hk/cardano-sl
# [CSE-254] Update dependencies of Explorer (front-end only)

~Side-note: With `post-css-button` we are sticking w/ `v.0.2.6` to avoid to have another feature which will break our styles~. ~See https://github.com/francoisromain/postcss-button/issues/1 for more information.~

_Task_: https://iohk.myjetbrains.com/youtrack/issue/CSE-254

_How to test:_
1. [Build](https://github.com/input-output-hk/cardano-sl/tree/master/explorer/frontend#short-version-of-installation) `Explorer` locally and run it with [mock server](https://github.com/input-output-hk/cardano-sl/tree/master/explorer#run-mock-server).
2. Open http://localhost:3100/ and check pages. Everything (styles etc.) should be the same as we had before.
